src/game.cpp

```diff
@@ -10147,13 +10179,37 @@ void game::butcher()
         break;
     case BUTCHER_CORPSE:
         {
+            uimenu smenu;
```

Comment:
Adding additional menu might cause player complaining about keypress, like the last time when hoe getting a digging function.

Reply:
Believe me, this is the minimal approach that could be done. I even reworked the initial commits to reduce the clutter. If you have a better idea how to make this choice without asking a player tell me.

Reply:
My idea is that the game should remember the butcher mode and player can switching between them.
So normal usage will be "Select mode -> butcher however many corpses you like -> Select another mode". 
Just like the gun firing mode.